src/hooks/subscription/CommunityUpdatesNotifier.tsx

```diff
@@ -0,0 +1,48 @@
+import { useApolloErrorHandler, useConfig, useNotification, useUserContext } from '..';
+import { useCommunicationUpdateMessageReceivedSubscription } from '../generated/graphql';
+import { FEATURE_COMMUNICATIONS, FEATURE_SUBSCRIPTIONS } from '../../models/constants';
+import { logger } from '../../services/logging/winston/logger';
+
+const useCommunityUpdatesNotifier = () => {
+  const { isFeatureEnabled } = useConfig();
+  const { isAuthenticated } = useUserContext();
+
+  if (!isFeatureEnabled(FEATURE_COMMUNICATIONS) || !isFeatureEnabled(FEATURE_SUBSCRIPTIONS) || !isAuthenticated) {
+    return;
+  }
+
+  try {
+    CommunityUpdatesSubscriber();
+  } catch (error) {
+    logger.error('[Updates Notifier] Failed subscribing for community updates. Failing gracefully.');
+  }
+};
+
+const CommunityUpdatesSubscriber = () => {
+  const handleError = useApolloErrorHandler();
+  const notify = useNotification();
+  const { user: userMetadata } = useUserContext();
+  const userId = userMetadata?.user.id;
+
+  useCommunicationUpdateMessageReceivedSubscription({
```

Comment:
hooks can be called only inside other hooks.
hooks can not be called conditionally.